pallets/vc-management/src/lib.rs

```diff
@@ -218,19 +239,45 @@ pub mod pallet {
 
 	#[pallet::call]
 	impl<T: Config> Pallet<T> {
+		/// add an account to the delegatees
 		#[pallet::call_index(0)]
+		#[pallet::weight(<T as Config>::WeightInfo::add_delegatee())]
+		pub fn add_delegatee(origin: OriginFor<T>, account: T::AccountId) -> DispatchResult {
+			let _ = T::DelegateeAdminOrigin::ensure_origin(origin)?;
+			// we don't care if `account` already exists
+			Delegatee::<T>::insert(account.clone(), ());
+			Self::deposit_event(Event::DelegateeAdded { account });
+			Ok(())
+		}
+
+		/// remove an account from the delegatees
+		#[pallet::call_index(1)]
```

Comment:
Why you push these calls at the beginning ?

Reply:
I wanted them to be consistent with IMP :(